### ISSUE TITLE
Provide class/class constant completion

### ISSUE THREAD

**Issue by Koc** [Haehnchen/idea-php-toolbox#24]
1. For some cases better add constant completion instead of plain strings (in this case we should import class maybe before completion):
    - `Symfony\Component\HttpFoundation\Response` https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/Response.php#L21
    - `Symfony\Component\Console\Output\OutputInterface` https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Output/OutputInterface.php#L55
2. Starting from php 5.5 we could use `::class` magic constant and better use this reference in some cases like behat context classes or form types. But maybe it should be configurable

**Comment by Haehnchen** [comment]
some examples `Request::isMethod` ` newRequest(null, <caret>)`. logic already implemented in FormType / getRepository. just a custom sorter is missing

**Comment by Koc** [comment]
Yes, in FormType it works ok. Maybe we should add some flag to json and plugin settings that it is possible import class as class and do not use plain strings.

**Comment by Koc** [comment]
Also it can be contributor for class constants - grep constants by prefix, like `PATHINFO_*` for http://ua2.php.net/manual/en/function.pathinfo.php

**Comment by Koc** [comment]
@Haehnchen is it possible create contributor for class constants? For example I want add autocomplete for `defined`/`constant` native PHP functions.

**Comment by Koc** [comment]
Also it is better suggest `FILE_*` constants for the 3rd argument of `file_put_contents` instead of other variables. Maybe push it to the top of the suggestions list.
